6-AdvancedScenarios/1-call-api-obo/API/fetch.js

```diff
@@ -0,0 +1,42 @@
+const msal = require('@azure/msal-node');
```

Comment:
should rename this file to something more accurate

Reply:
Done!